apmpackage/apm/changelog.yml

```diff
@@ -3,6 +3,14 @@
 # change type can be one of: enhancement, bugfix, breaking-change
 - version: "generated"
   changes:
+    - description: Placeholder
+      type: bugfix
+      link: https://github.com/elastic/apm-server/pull/123
+- version: "8.6.0"
```

Comment:
**question**: It seems the 8.6 changelog was missing from this branch. Is that intended ?

Reply:
I believe so, since the changelog is generated when we generate the package and the "generated" string gets replaced by the actual version.

Reply:
Ah, 8.6 is not released yet :facepalm: 

Thanks!